tutorials/colors/colorbar_only.py

```diff
@@ -38,6 +38,26 @@
 fig.colorbar(mpl.cm.ScalarMappable(norm=norm, cmap=cmap),
              cax=ax, orientation='horizontal', label='Some Units')
 
+
+###############################################################################
+# Extended colorbar with continuous colorscale
+# --------------------------------------------
+
```

Comment:
```suggestion
#
```
if you want the below comment to be part of the text in the rendered file, instead of a comment.

Reply:
Done.